components/dropdown/dropdown-content-mixin.js

```diff
@@ -687,6 +702,18 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		await adjustPosition();
 	}
 
+	__startObservingContent() {
+		if (this.__mutationObserver) this.__mutationObserver.disconnect();
+		this.__mutationObserver = new MutationObserver(this.__position);
+		const slottedNodes = this.__getContentSlottedNodes();
+		for (let i = 0; i < slottedNodes.length; ++i) {
+			this.__mutationObserver.observe(slottedNodes[i], {
```

Comment:
I'm not sure about the performance impacts of this idea - it'll re-render whenever the contents change. For example, the filter adds the dimensions one-by-one when they are loaded asynchronously. This will cause us to run position every time.